app/phpunit.xml.dist

```diff
@@ -22,22 +32,6 @@
         <log type="junit" target="../build/logs/junit.xml" logIncompleteSkipped="false"/>
     </logging>
 
-    <php>
```

Comment:
Why is this removed?

Reply:
Most of it was actually just moved up in the file.